Branch-SDK/src/main/java/io/branch/coroutines/DeviceSignals.kt

```diff
@@ -0,0 +1,52 @@
+package io.branch.coroutines
+
+import android.content.Context
+import android.webkit.WebSettings
+import android.webkit.WebView
+import io.branch.referral.BranchLogger.e
+import io.branch.referral.BranchLogger.v
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+
+
+/**
+ * Returns the user agent string on a background thread via static class WebSettings
```

Comment:
Explain why we have two methods and why we'd use over the other.

Reply:
Added comments